packages/app/src/app/overmind/effects/live/index.ts

```diff
@@ -495,6 +495,18 @@ class Live {
     });
   }
 
+  reset() {
+    this.clients.clear();
+
+    if (this.awaitSendTimer) {
+      clearTimeout(this.awaitSendTimer);
+    }
```

Comment:
```suggestion
    clearTimeout(this.awaitSendTimer);
```

No need to check :-)